_chapters/modularity.md

```diff
@@ -1,3 +0,0 @@
----
-title: Modular Design
----
```

Comment:
Were these just empty? I can't seem to find anything that looks like it corresponds on the current site?

Reply:
Yes they were - https://github.com/18F/frontend/blob/master/_chapters/modularity.md